demo/.eslintrc.js

```diff
@@ -14,8 +14,9 @@ module.exports = {
     "prettier/react",
   ],
   rules: {
-    "no-console": 0,
-    "react/prop-types": 0,
+    "no-console": "off",
+    "react-native/no-raw-text": "off",
```

Comment:
I think we want `react-native/no-raw-text` enabled in the long run. But either way, this change to strings for the config rather than integers is a great change!

https://github.com/Intellicode/eslint-plugin-react-native/blob/master/docs/rules/no-raw-text.md

Reply:
This can wait until a future issue, though https://github.com/tasitlabs/tasit/issues/28